games-emulation/retroarch/retroarch-1.7.1.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="The reference frontend of libretro"
+DESCRIPTION="RetroArch is a frontend for emulators, game engines and media \
+players. It enables you to run classic games on a wide range of computers and \
+consoles through its slick graphical interface. It has advanced features like \
+shaders, netplay, rewinding, next-frame response times, and more!"
+HOMEPAGE="https://libretro.com/"
+COPYRIGHT="2010-2018 The RetroArch Team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/libretro/RetroArch/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="e9189c7da6b2ca0142528eca14f26104ee322f052f8609203d90b60938d8d3f3"
+SOURCE_FILENAME="retroarch-$portVersion.tar.gz"
+SOURCE_DIR="RetroArch-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	$relativeSettingsDir/retroarch.cfg auto-merge
+	"
+
+USER_SETTINGS_FILES="
+	$relativeSettingsDir/retroarch.cfg template \
+		$relativeDataDir/retroarch/retroarch.cfg.default
+	"
+
+PROVIDES="
+	retroarch$secondaryArchSuffix = $portVersion
+	app:retroarch$secondaryArchSuffix = $portVersion
+	cmd:retroarch$secondaryArchSuffix = $portVersion
+	cmd:retroarch_cg2glsl$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix
+	lib:libavformat$secondaryArchSuffix
+	lib:libavdevice$secondaryArchSuffix
+	lib:libswresample$secondaryArchSuffix
+	lib:libavresample$secondaryArchSuffix
+	lib:libavutil$secondaryArchSuffix
+	lib:libswscale$secondaryArchSuffix
+	retroarch_assets
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsdl2$secondaryArchSuffix
+	devel:libgl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:git
```

Comment:
Good catch, git is not required anymore as we're not running the fetch-submodules script.

Reply:
Good catch.
If it builds without `cmd:git` (and **_without_** passing `--no-git-repo` to haikuporter) then it would be nice to drop the `cmd:git` here.

(I don't know if that's the case here, but some recipes like dev-vcs/guitar fail to build without `cmd:git` because their build process looks for a `.git` directory, and if it finds that directory but cannot find git itself, it fails to build. This could be fixed by having the builders pass `--no-git-repo` to haikuporter to prevent the creating of the `.git` directory. Until then, we need to keep the _not-really-needed_ `cmd:git` in these recipes.)